BigBaseV2/src/views/self/view_self.cpp

```diff
@@ -158,6 +159,58 @@ namespace big
 			}
 		}
 
+		ImGui::Separator();
+
+		components::sub_title("HUD");
+
+		ImGui::BeginGroup();
+
+		ImGui::Checkbox("Hide Radar", &g->self.hide_radar);
+
+		ImGui::SameLine();
+
+		ImGui::Checkbox("Hide Ammo", &g->self.hide_ammo);
+
+		ImGui::SetNextItemWidth(500.f);
+		ImGui::SliderInt(
+			"##hud_comp_slider",
+			&g->self.selected_hud_component,
+			1,
+			(int)HudComponents::HUD_WEAPONS,
+			(
+				g->self.selected_hud_component >= 1 &&
+				g->self.selected_hud_component <= (int)HudComponents::HUD_WEAPONS
+			) ? hud_component_names[g->self.selected_hud_component] : hud_component_names[0]
+		);
```

Comment:
Weird way of drawing something that could be a ListBox or ComboBox.

Please change to one of these instead.

Reply:
I was gonna make it a combo box, but made it a slider instead because I thought it would take up too much of the screen. Will change nonetheless.

Reply:
Done. Made it a ComboBox. Sorry for being late, I was at school practically all day.